tgui/packages/tgui/interfaces/Reagents.js

```diff
@@ -0,0 +1,418 @@
+import { useBackend, useLocalState } from '../backend';
+import { Button, LabeledList, Section, Table, Icon, Stack } from '../components';
+import { Window } from '../layouts';
+import { map } from 'common/collections';
+import { RecipeLookup } from './common/RecipeLookup';
+import { ReagentLookup } from './common/ReagentLookup';
+import { logger } from '../logging';
+
+/* functions */
+
+const bookmarkedReactions = new Set();
+
+const matchBitflag = (a, b) => (a & b) && (a | b) === b;
+
+const hasReagentType = (currentReagents, reagent) => {
+  if (currentReagents === null) {
+    return false;
+  }
+  if (currentReagents.includes(reagent)) {
+    return true;
```

Comment:
you can replace all of this function with currentReagents?.includes(reagent)

Reply:
done